src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -80,6 +80,11 @@ public function createAction(Request $request): Response
 
             return $this->redirectToRoute('admin_product_new');
         }
+        if ($this->get('prestashop.adapter.multistore_feature')->isUsed()) {
+            $this->addFlashMessageNotMultistoreCompatible();
+
+            return $this->render('@PrestaShop/Admin/Sell/Catalog/Product/disabled.html.twig');
```

Comment:
You only handed this for new product page (createAction) but you also (mostly) need to handle it for the edit page (editAction)